portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+import org.jetbrains.annotations.Nullable;
+
+public class Comment{
+    //personal data
+    final private String firstName, lastName, email, phone;
+    //job related data: optional
+    @Nullable private String jobTitle = null;
```

Comment:
Nit: initializing to null is not needed.

Reply:
done on bori_dev_2